### ISSUE TITLE
Table borders are disappeared after saving ckeditor data in my application . Just text is visible.

### ISSUE THREAD

**Issue by Tiru150837** [ckeditor/ckeditor5#14229]
### Origin URL

https://ckeditor.com/docs/ckeditor5/latest/features/tables

### Project version

28.0.0

### Is there anything else you would like to add?

Our application is very old which is using Angular 5.1 version .  So I try to use the minor versions of CKEditor5. CKEditor5-angular version is 1.2.3.  And classic build version is 28.0.0. While build and compiling , I didn't get any error.  But I try to use the table and I entered some data in it. After saving this in my application , the only data is visible and borders are not visible.  But I didn't see any errors in my Network tab also. What should I do?

Before we were using CKEditor4 . So I am trying to upgrade the version CKEditor4 to CKEditor5. In CKEditor4 that issue is not coming.  Is there any suggestions can helpful to me?

### User agent

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.50

**Comment by Reinmar** [comment]
You'll find information on how to style the content in [https://ckeditor.com/docs/ckeditor5/latest/installation/advanced/content-styles.html](https://ckeditor.com/docs/ckeditor5/latest/installation/advanced/content-styles.html)

tl;dr is – you need to apply a stylesheet (e.g the one proposed in that guide) to the output of the editor when displaying it somewhere else

**Comment by Tiru150837** [comment]
Can I need to create own stylesheet ?

But how can I use that specific stylesheet to specidfic item like table ?

**Comment by Tiru150837** [comment]
Can I directly applied to my ckeditor component ?

**Comment by Tiru150837** [comment]
Thank you Reinmar. Now the problem is resolved
